### PR TITLE
Hive 23690 : TestNegativeCliDriver#[external_jdbc_negative] is flaky

### PR DESCRIPTION
**Pull request by rkirtir**
### What changes were proposed in this pull request?
Reenabling external_jdbc_negative.q 

### Why are the changes needed?
http://130.211.9.232/job/hive-flaky-check/34/

### Does this PR introduce _any_ user-facing change?
NO

### Is the change a dependency upgrade?
NO

### How was this patch tested?
with q file
Flakey run test http://130.211.9.232/job/hive-flaky-check/765/

### CODE REVIEW

**Comment by rkirtir** [comment]
NPE in JDBC storage handler has been fixed with https://issues.apache.org/jira/browse/HIVE-27487
https://github.com/apache/hive/pull/4475

With this commit JdbcSerDe.java has littile change in SerDeException message so .q.out file has been overwritten

**Comment by rkirtir** [comment]
@ayushtkn  Requesting your review

File: ql/src/test/queries/clientnegative/external_jdbc_negative.q [hunk 1]
```diff
@@ -1,6 +1,5 @@
---! qt:disabled:test is unstable HIVE-23690
 --! qt:dataset:src
-
+set user.name=user1;
```
[the remark this answers is not in the document]
Comment (rkirtir) [reply ql/src/test/queries/clientnegative/external_jdbc_negative.q:2]: Agree. Even I was skeptical about seeting this. But had a flakey build stating 
" Schema 'USER1' does not exist)"

http://ci.hive.apache.org/job/hive-flaky-check/764/

I look forward to suggestion

[the remark this answers is not in the document]
Comment (rkirtir) [reply ql/src/test/queries/clientnegative/external_jdbc_negative.q]: fixed and here are flakey job tests

http://ci.hive.apache.org/job/hive-flaky-check/775/
http://ci.hive.apache.org/job/hive-flaky-check/774/

Seems flakey job itself was flakey :)

**Review by InvisibleProgrammer — APPROVED** [review]
LGTM.

I have just one request: after the previous conversion about `set user.name=user1;` with @aturoczy , that line has been removed. Could you please rerun the flaky test job as the test changed since the last run?

**Comment by rkirtir** [comment]
I have reran flakey job 2 times after reverting set user

http://ci.hive.apache.org/job/hive-flaky-check/775/
http://ci.hive.apache.org/job/hive-flaky-check/774/

**Review by veghlaci05 — APPROVED** [review]
LGTM

**Pull request closed without merging**